packages/design-system/src/components/OcCheckbox/OcCheckbox.vue

```diff
@@ -84,14 +84,6 @@ export default defineComponent({
       type: Boolean,
       default: false
     },
-    /**
-     * Is the label of the Checkbox visually hidden?
-     **/
-    hideLabel: {
-      type: Boolean,
-      required: false,
-      default: false
-    },
```

Comment:
Why did you remove this, didn't you just add it recently?

Reply:
I added labelHidden prop, which is more a11y compatible than the implementation behind the hideLabel prop. I adjusted the other occurrences